### PR TITLE
Add 3rd-party dependencies registration topic

### PR DESCRIPTION
**Pull request by GoshaFighten**

### CODE REVIEW

File: docs/setup-3rd-party-dependencies.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# Third-party dependencies registration in Angular CLI 6+
```
Comment (RomanTsukanov) [inline docs/setup-3rd-party-dependencies.md]: Title Capitalization?

Comment (GoshaFighten) [reply docs/setup-3rd-party-dependencies.md]: https://github.com/DevExpress/devextreme-angular/blob/master/docs/using-angular-cli.md

The same is here. I copied the existing style.

Comment (RomanTsukanov) [reply docs/setup-3rd-party-dependencies.md]: In the document you linked, all headers except "Create a new Application" actually use the Title Case.

Comment (GoshaFighten) [reply docs/setup-3rd-party-dependencies.md]: Fixed

**Pull request merged**